src/helpers/copyContract.js

```diff
@@ -0,0 +1,55 @@
+const RECEIPT_CHECK_TIMEBOX = 100
+const DEPLOYMENT_TIMEOUT = 60000
+const DEPLOYMENT_TIMEOUT_ERROR = 'Contact deployment timed out'
+
+function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms))
+}
+
+function callback(resolve, reject) {
+  return (error, result) => error ? reject(error) : resolve(result)
+}
+
+function sendTransaction(params) {
+  return new Promise(function (resolve, reject) {
+    web3.eth.sendTransaction(params, callback(resolve, reject))
+  })
+}
+
+async function getTransactionReceipt(txHash) {
+  for (let i = 0; i < (DEPLOYMENT_TIMEOUT / RECEIPT_CHECK_TIMEBOX) ; i++) {
```

Comment:
Note that this loop will end well past the deployment timeout, since it's not accounting for how long the body of the for-loop runs (and getTransactionReceipt may be quite lengthy). 

If we consider this to be an issue (it may very well not be), we can either:
- Use a lodash-debounced version of the function, where we call back into the function to iterate, and don't need to manually `sleep` on every iteration (extra points because lodash!)
- Run the for-loop every TIMEBOX ms, but abort after TIMEOUT ms have passed, and not after TIMEOUT/TIMEBOX iterations.